recipe/meta.yaml

```diff
@@ -1,32 +1,39 @@
-{% set version = "3.06.1" %}
+{% set version = "3.08" %}
 
 package:
   name: nsis
   version: {{ version }}
 
 source:
   - url: http://sourceforge.net/projects/nsis/files/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}.zip
-    sha256: d463ad11aa191ab5ae64edb3a439a4a4a7a3e277fcb138254317254f7111fba7
+    sha256: 1bb9fc85ee5b220d3869325dbb9d191dfe6537070f641c30fbb275c97051fd0c
     folder: binary
-
   - url: http://sourceforge.net/projects/nsis/files/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}-src.tar.bz2  # [unix]
-    sha256: 9b5d68bf1874a7b393432410c7e8c376f174d2602179883845d2508152153ff0  # [unix]
+    sha256: a85270ad5386182abecb2470e3d7e9bec9fe4efd95210b13551cb386830d1e87  # [unix]
     folder: src   # [unix]
     patches:  # [unix]
       - 0001-Always-use-relative-paths-for-CONF-and-DATA-directories.patch  # [unix]
-
+  # Plugins
+  - url: https://nsis.sourceforge.io/mediawiki/images/6/62/BgWorker.zip
+    sha256: be8c5264c796309c465d017b12e12aee62d3c8c20966bbbecb96fb8fabb023ce
+    folder: plugins/BgWorker
   - url: https://code.kliu.org/misc/elevate/elevate-1.3.0-redist.7z
     sha256: b1b3f070353a0eadee2cea3a575049d10df9763ff24e39313da4cec9455382e1
     folder: plugins/elevate
-  - url: http://nsis.sourceforge.net/mediawiki/images/7/79/UAC_v0.2.2d.zip
-    sha256: 9e64d93185e468fb873925db887f637778d926b864b5ff85600b7c9fce92660d
+  - url: https://nsis.sourceforge.io/mediawiki/images/8/8f/UAC.zip
+    sha256: 20e3192af5598568887c16d88de59a52c2ce4a26e42c5fb8bee8105dcbbd1760
     folder: plugins/UAC
   - url: https://github.com/mingwandroid/nsis-untgz/archive/5c814c5f2c8a9a14e3a6ddd3e594fcc76db5b86a.zip
     sha256: ae68c41493abbb8800640acdf67a06c63bcceaaf21b539c50c348a20dc4b2803
     folder: plugins/untgz
   - url: https://github.com/mingwandroid/nsis-UnicodePathTest/archive/fa74caef553883f1820049d89e169aff57551796.zip
     sha256: 1a4dc09f0fbb7d6be88835c50a7c95dbd37470c65e29cf676b14e3ad9c4d7494
     folder: plugins/UnicodePathTest
+  # AccessControl does not provide historical versions as far as I can tell..
+  # this is for version v1.0.8.3 - 24th February 2021
+  - url: https://nsis.sourceforge.io/mediawiki/images/4/4a/AccessControl.zip
```

Comment:
Please upload this archive somewhere and use that instead of using an unstable link like this